core/deployment/pom.xml

```diff
@@ -38,12 +38,6 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-bootstrap-core</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.eclipse.sisu</groupId>
-                    <artifactId>org.eclipse.sisu.inject</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
Hmmm. I'm a bit nervous about that one. This particular artifact generates a file at the root and we excluded it to avoid that.

Reply:
See https://github.com/quarkusio/quarkus/pull/2986 .